<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1460</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="outGBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>1391</width>
      <height>551</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>OutPut</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="outTable">
       <column>
        <property name="text">
         <string>Src Addr</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Des Addr</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Src MAC</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Des MAC</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Src Port</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Des Port</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Protocol</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Info</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>620</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>620</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>620</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addrSLEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>600</y>
      <width>161</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="placeholderText">
     <string>Set IP Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="sourceLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>600</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Source:</string>
    </property>
    <property name="buddy">
     <cstring>addrSLEdit</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="destLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>660</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Destination:</string>
    </property>
    <property name="buddy">
     <cstring>addrDLEdit</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="addrDLEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>660</y>
      <width>161</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Set IP Address</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1460</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>919</x>
     <y>663</y>
    </hint>
    <hint type="destinationlabel">
     <x>854</x>
     <y>705</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
